modules/REST/V1/Options.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\REST\V1;
+
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+use WP_Syntex\Polylang\Options\Options as Options_Registry;
+use WP_Syntex\Polylang\REST\Abstract_Controller;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Options REST controller.
+ *
+ * @since 3.7
+ */
+class Options extends Abstract_Controller {
+	/**
+	 * @var Options_Registry
+	 */
+	private $options;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 3.7
+	 *
+	 * @param Options_Registry $options Options registry.
+	 */
+	public function __construct( Options_Registry $options ) {
+		parent::__construct();
+		$this->rest_base = 'options';
+		$this->options   = $options;
+	}
+
+	/**
+	 * Registers the routes for options.
+	 *
+	 * @since 3.7
+	 *
+	 * @return void
+	 */
+	public function register_routes(): void {
+		register_rest_route(
+			$this->namespace,
+			"/{$this->rest_base}",
+			array(
+				'args'        => array(
+					'context' => $this->get_context_param( array( 'default' => 'view' ) ),
```

Comment:
I'd use `array( 'default' => 'edit' )` instead, what do you think?

Reply:
So we don't need to pass it by default? Why not, good idea.

Reply:
Done in [2e04b6e](https://github.com/polylang/polylang/pull/1569/commits/2e04b6e45506c889bbc7b39c45072e3814c4d395).